esmvaltool/preprocessor/__init__.py

```diff
@@ -47,8 +53,14 @@
     'mask_fillvalues',
     # Region selection
     'extract_region',
+    'extract_volume',
+    'extract_trajectory',
+    'extract_transect',
     # Grid-point operations
+    'depth_integration',
     'average_region',
+    'average_volume',
+    'time_average',
     # 'average_zone': average_zone,
```

Comment:
For the reasons outlined in the discussion, I think this should be moved after `multi_model_statistics`, to keep the processing chain as general as possible.